### PR TITLE
Cluster Scoped Runthrough

### PR DESCRIPTION
**Pull request by cwilkers**
Copied README-ocp4.md into a cluster scoped version and modified to work with operator in cluster scoped mode.

### CODE REVIEW

**Comment by cooktheryan** [comment]
@cwilkers can you rebase and potentially add to the README once #33 is merged

File: README-ocp4-cs.md [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+<a id="markdown-federation-deployment" name="federation-deployment"></a>
+# Federation deployment
+
+<a id="markdown-create-the-two-openshift-clusters" name="create-the-two-openshift-clusters"></a>
+## Create the two OpenShift clusters
+
+Follow the [Installer Provisioned Infrastructure Instructions](https://cloud.redhat.com/openshift/install/aws/installer-provisioned) for installing two OpenShift 4.1 clusters on AWS. Be sure to employ the `--dir` option with a distinct directory for each cluster you create.
+
+<a id="markdown-configure-client-context-for-cluster-admin-access" name="configure-client-context-for-cluster-admin-access"></a>
+## Configure client context for cluster admin access
```
Comment (mvazquezc) [inline README-ocp4-cs.md]: What do you think about referencing your video about flattening kubeconfig files? 

https://www.youtube.com/watch?v=zuh3JFtNbeU

File: README-ocp4-cs.md [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+
+~~~
+
+Now we are going to enable some of the federated types needed for our demo
+application. You can watch as FederatedTypeConfig resources are created for
+each type by visiting the `All Instances` tab of the `Kubefed Operator` under
+`Catalog -> Installed Operators`.
+
+~~~sh
+for type in namespaces secrets serviceaccounts services configmaps deployments.apps
```
Comment (mvazquezc) [inline README-ocp4-cs.md]: In order to demonstrate the cluster scoped federation, we should enable a cluster-wide type like ClusterRole.

What do you think about enabling a ClusterRole and then creating a FederatedClusterRole within the demo-app?

Comment (cwilkers) [reply README-ocp4-cs.md]: Yes! I forgot I wanted to add this before merging. I'll put the WIP label back on.

**Comment by scollier** [comment]
I walked through this up to here: https://github.com/openshift/federation-dev/blob/b253f1fa6df73254e178c15a712b27fa987c640b/README-ocp4-cs.md#deploy-federation  and it is a /lgtm from there.

**Comment by mvazquezc** [comment]
@cwilkers What do you think about creating a custom scc that allows `anyuid` for the ngincs serviceaccount? Since SCCs are cluster scoped objects we might use a FederatedScc to demonstrate cluster scoped items propagation.

**Comment by cwilkers** [comment]
That's a great idea, I was trying to figure out how to create the scc as a
federate able object, but it doesn't look possible.

**Comment by mvazquezc** [comment]
After the latest changes lgtm. Let me know when you want me to merge.

**Pull request merged**